packages/lib/src/utils/Transactions.js

```diff
@@ -32,7 +36,7 @@ const TRUFFLE_DEFAULT_GAS_PRICE = BN(100000000000);
  * @param retries number of transaction retries
  */
 export async function sendTransaction(contractFn, args = [], txParams = {}, retries = RETRY_COUNT) {
-  await checkGasPrice(txParams);
+  txParams = await fixGasPrice(txParams)
```

Comment:
`fixGasPrice` is both modifying `txParams` and returning it, which may be confusing to users of the function. Let's either: 1- remove the return value, and just call it as `await fixGasPrice(txParams)` (without the need for assigning to txParams)
2- or have fixGasPrice return a copy of txParams with gasPrice set (without modifying the parameter, so you'd need to make a copy of the txParams parameter among the first things you do in fixGasPrice)